Fix the ALCvoid reference to make blender and a bunch of other
programs work.

diff -Naur openal-0.0.8/common/include/AL/alc.h openal-0.0.8.new/common/include/AL/alc.h
--- openal-0.0.8/common/include/AL/alc.h	2006-02-11 10:36:55.000000000 +0100
+++ openal-0.0.8.new/common/include/AL/alc.h	2008-03-31 17:42:09.000000000 +0200
@@ -187,7 +187,7 @@
 
 ALC_API void            ALC_APIENTRY alcDestroyContext( ALCcontext *context );
 
-ALC_API ALCcontext *    ALC_APIENTRY alcGetCurrentContext( ALCvoid );
+ALC_API ALCcontext *    ALC_APIENTRY alcGetCurrentContext( ALCvoid *buffer );
 
 ALC_API ALCdevice*      ALC_APIENTRY alcGetContextsDevice( ALCcontext *context );
 
@@ -248,7 +248,7 @@
 typedef void           (ALC_APIENTRY *LPALCPROCESSCONTEXT)( ALCcontext *context );
 typedef void           (ALC_APIENTRY *LPALCSUSPENDCONTEXT)( ALCcontext *context );
 typedef void           (ALC_APIENTRY *LPALCDESTROYCONTEXT)( ALCcontext *context );
-typedef ALCcontext *   (ALC_APIENTRY *LPALCGETCURRENTCONTEXT)( ALCvoid );
+typedef ALCcontext *   (ALC_APIENTRY *LPALCGETCURRENTCONTEXT)( ALCvoid *buffer );
 typedef ALCdevice *    (ALC_APIENTRY *LPALCGETCONTEXTSDEVICE)( ALCcontext *context );
 typedef ALCdevice *    (ALC_APIENTRY *LPALCOPENDEVICE)( const ALCchar *devicename );
 typedef ALCboolean     (ALC_APIENTRY *LPALCCLOSEDEVICE)( ALCdevice *device );
diff -Naur openal-0.0.8/src/alc/alc_context.c openal-0.0.8.new/src/alc/alc_context.c
--- openal-0.0.8/src/alc/alc_context.c	2006-01-23 18:01:13.000000000 +0100
+++ openal-0.0.8.new/src/alc/alc_context.c	2008-03-31 17:45:32.000000000 +0200
@@ -1083,7 +1083,7 @@
  * suitable for use with every function that takes a context handle,
  * or NULL if there is no current context.
  */
-ALCcontext *alcGetCurrentContext( void )
+ALCcontext *alcGetCurrentContext( ALCvoid *buffer )
 {
 	if(al_contexts.items == 0)
 	{
